### DISCUSSION TITLE
What are the long-term goals for this project?

### DISCUSSION THREAD

**Discussion by totkeks** [LibreHardwareMonitor/LibreHardwareMonitor#1271] [General]
Hello @PhyxionNL and contributors.

I found this fork a while ago and was wondering two things: For one, what are your long-term goals for this project and the other were, how can I contribute.

Have you ever tried to contact the original author of `OpenHardwareMonitor` about taking over his project and app name officially? I personally find a bit confusing that both projects co-exist, even though the `OHM` is not maintained anymore.

Regarding goals, is there any issue that's tracking a kind of roadmap?

From what I have seen when looking through the code, a documentation / architecture description or diagram could be really helpful. Also updating to a newer .NET version, like 8 or 9 could be a good idea.

And then maybe an installer, packaging for winget / Windows store.

What do you think about this?

**Comment by visormail** [comment]
Additionally there is another fork https://github.com/hexagon-oss/openhardwaremonitor and that contains the Report component too.
